webapp/packages/core-app/src/AppScreen/RightArea.tsx

```diff
@@ -6,46 +6,35 @@
  * you may not use this file except in compliance with the License.
  */
 import { observer } from 'mobx-react-lite';
-import styled, { css } from 'reshadow';
 
 import {
   Loader,
   Pane,
   ResizerControls,
+  s,
   SlideBox,
-  slideBoxStyles,
+  SlideBoxStyles,
   SlideElement,
   SlideOverlay,
   Split,
-  splitHorizontalStyles,
-  splitStyles,
+  SplitHorizontalStyles,
+  SplitStyles,
```

Comment:
Please use this style in Split controls instead; you can determine splitter type by `split="horizontal"` prop